react-components/src/components/Widgets/WindowWidget.tsx

```diff
@@ -54,6 +54,9 @@ export const WindowWidget = ({
   }
 
   const handleExpand = (): void => {
+    if (!isMinimized) {
+      setPosition({ x: 0, y: 0 });
```

Comment:
I'm not sure what this does :thinking: Does it put the window back to some specific position on screen every time it's expanded?

Reply:
It places the window back to a specific position when minimized